packages/ethereum/contracts/contracts-addresses.json

```diff
@@ -55,10 +55,10 @@
       "indexer_start_block_number": 24097267,
       "network_registry_contract_address": "0x819E6a81e1e3f96CF1ac9200477C2d09c676959D",
       "network_registry_proxy_contract_address": "0xcA9B1bC189F977B2A9217598D0300d956b6a719f",
-      "stake_contract_address": "0x65c39e6BD97f80B5aE5d2120a47644578Fd2B8DC",
-      "stake_season": 7,
+      "stake_contract_address": "0x488EB03650BD2de6b4fEB187F06b34Ee483D81ba",
```

Comment:
This should be 0xdc8f03f19986859362d15c3d5ed74f26518870b9 after the most recent contract changes?

Reply:
forgot to push changes in this branch. It's updated now